### PR TITLE
Backwards compatible cleanup

### PR DESCRIPTION
**Pull request by simonmika**

### CODE REVIEW

File: Identifier.ts [hunk 1]
```diff
@@ -83,46 +80,18 @@ export namespace Identifier {
...
+	export type Length = typeof Length.values[number]
 	export namespace Length {
-		export function is(value: Length | any): value is Length {
-			return typeof value == "number" && value >= 4 && value <= 128 && (value & 252) == value
-		}
+		export const values = [
+			4, 8, 12, 16, 20, 24, 28, 32, 36, 40, 44, 48, 52, 56, 60, 64, 68, 72, 76, 80, 84, 88, 92, 96, 100, 104, 108, 112,
+			116, 120, 124, 128,
+		] as const
+		export const type = isly.named("cryptly.Identifier", isly.number<Length>(values as any as number[]))
```
Comment (SiFalorSum) [inline Identifier.ts]: Should the name be `"cryptly.Identifier.Length"` instead?

File: Digester/Algorithm.ts [hunk 2]
```diff
@@ -1,7 +1,10 @@
-export type Algorithm = "SHA-1" | "SHA-256" | "SHA-384" | "SHA-512"
+import { isly } from "isly"
+
+export type Algorithm = typeof Algorithm.values[number]
 
 export namespace Algorithm {
-	export function is(value: Algorithm | any): value is Algorithm {
-		return value == "SHA-1" || value == "SHA-256" || value == "SHA-384" || value == "SHA-512"
-	}
+	export const values = ["SHA-1", "SHA-256", "SHA-384", "SHA-512"]
```
Comment (Magnus93) [inline Digester/Algorithm.ts]: `as const` otherwise `Algorithm` will just be type `string`.

Comment (Magnus93) [reply Digester/Algorithm.ts]: Also the array can then technically be changed.

File: Identifier.ts (hunk 1 above)
Comment (simonmika) [reply Identifier.ts]: Thanks

File: Digester/Algorithm.ts (hunk 2 above)
Comment (simonmika) [reply Digester/Algorithm.ts]: Thanks

File: package.json [hunk 3]
```diff
@@ -69,16 +69,18 @@
...
-		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20231004",
-		"eslint-plugin-simple-import-sort": "^12.0.0",
+		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20340516",
+		"eslint-plugin-simple-import-sort": "^12.1.0",
 		"jest": "^29.7.0",
-		"prettierx": "github:utily/prettierx#utily-20231004",
-		"rimraf": "^5.0.5",
-		"ts-jest": "^29.1.2",
-		"typescript": "^5.3.3"
+		"prettierx": "github:utily/prettierx#utily-20340516",
```
Comment (SiFalorSum) [inline package.json]: Wrong tag??

Comment (simonmika) [reply package.json]: Thanks

**Review by SiFalorSum — APPROVED** [review]

**Pull request merged**